src/chimera/interfaces/telescope.py

```diff
@@ -417,6 +417,28 @@ def unparkComplete(self):
         successfully.
         """
 
+    def openCover(self):
```

Comment:
@wschoenell: don't you think that it is better to have a new interface for this (TelescopeCover)? Looks like most scopes can park/unpark but will not have a cover.